docs/en-us/dev/user_doc/guide/task/python.md

```diff
@@ -1,15 +1,55 @@
 # Python Node
 
-- Using python nodes, you can directly execute python scripts. For python nodes, workers use `python **` to submit tasks.
+## Overview
 
-> Drag from the toolbar ![PNG](https://analysys.github.io/easyscheduler_docs_cn/images/toolbar_PYTHON.png) task node to the canvas, as shown in the following figure:
+`Python Task` is used to create a python-type task and execute python scripts. When the worker execute `Python Task`,
+a temporary python script is generated, and the Linux user with the same name as the tenant executes the script.
```

Comment:
', and the Linux user with the same name as the tenant executes the script.' 
should be 
', and executes the script by the Linux user with the same name as the tenant .'